README.md

```diff
@@ -1,6 +1,6 @@
-# Q# language and tooling
+# Modern QDK
```

Comment:
I think the term 'Modern' will be short lived, and fade into obscurity once the 'classic' QDK is forgotten about. I'd go with a heading of just 'Azure Quantum Development Kit'.